components/EditorialStory/index.tsx

```diff
@@ -7,12 +7,13 @@ interface IEditoralStory {
 
 const EditoralStory = ({ posts }: IEditoralStory) => {
   const [firstPost, ...columnPosts] = posts;
+  const secondaryPost = columnPosts.slice(1, 4);
   return (
     <>
       <div className="editorialStory">
         <PostPreview post={firstPost} variant="front" />
         <div className="columnPosts">
-          {columnPosts.map((element, key) => (
+          {secondaryPost.map((element, key) => (
```

Comment:
Why was this change needed? 🤔

Reply:
Because of the naming and the `secondaryPost` on the component only should display the 3 posts at the time.